.github/quarkus-github-bot.yml

```diff
@@ -780,7 +780,16 @@ triage:
       title: "(virtual.thread|RunOnVirtualThread)"
       notify: [cescoffier, ozangunalp]
     - id: podman
-      labels: [env/podman]
+      labels: [env/podman, area/container-image]
       title: "podman"
+      notify: [edeandrea]
+      directories:
+        - extensions/container-image
+    - id: docker
+      labels: [area/container-image]
+      title: "docker"
+      nodify: [edeandrea]
+      directories:
+        - extensions/container-image
```

Comment:
I'm not exactly excited about this change. For me issues with Podman and the container image extensions are orthogonal.

Reply:
Perhaps you are right, but for example I would feel comfortable working with the podman/docker extensions, but maybe not so with the others because they are each different. 

But I totally get it if the intent is to keep things higher level.

Reply:
I've updated the PR according to feedback.